java-frontend/src/test/java/org/sonar/java/model/JParserTest.java

```diff
@@ -96,13 +97,22 @@ void should_throw_RecognitionException_in_case_of_lexical_error() {
   @Test
   void err() {
     try {
-      // ASTNode.METHOD_DECLARATION with flag ASTNode.MALFORMED
+      // ASTNode.METHOD_DECLARATION with flag ASTNode.MALFORMED: missing return type
       test("interface Foo { public foo(); // comment\n }");
       fail("exception expected");
```

Comment:
I see, it's not from your code. But it's not the best way to assert exceptions and as you've already touched this test, you can replace it with `assertThrows()`